src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person: {
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+} }) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age === undefined ? (
+        null
+      ) : (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {!isMarried ? (
+          'I am not married'
+        ) : (
+          `${partnerName} is my ${sex === 'f' ? (
+            'husband'
+          ) : (
+            'wife'
+          )}`
+        )}
```

Comment:
nested ternary is not a good idea, create variable and move there this inside condition